netstandard/ref/System.IO.cs

```diff
@@ -447,11 +463,11 @@ public partial class FileStream : System.IO.Stream
         public virtual System.IntPtr Handle { get { throw null; } }
         public virtual bool IsAsync { get { throw null; } }
         public override long Length { get { throw null; } }
-        public string Name { get { throw null; } }
+        public virtual string Name { get { throw null; } }
```

Comment:
Should this be noted in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md#members?

> ✗ Disallowed
> 
> - Adding `virtual` to a member

Reply:
@jnm2 adding it as disallowed is a little stronger then we generally want as we generally allow this type of change. We just need to be caution when we do so.

Reply:
@weshaggard The document *currently* disallows—should we move it to a section called “⚠ Discouraged” or say something like that?

Reply:
That breaking-change-rules document is missing the nuances for many of the changes. It was originally started as the list of the 100% breaking changes that should be never done. Most  changes added since the original revision are the more subtle ones with nuances.

We did have a very long document describing the various types of breaking changes with nuances. It actually [publicly available for a while](https://blogs.msdn.microsoft.com/bclteam/2005/04/05/net-and-compatibility-breaking-changes-in-a-managed-world-kit-george/). I am not able to find it. Perhaps we may want to try to resurrect it instead of reinventing the wheel.

IIRC, the doc was actual called "Defining An Acceptable Change". Notice the focus on what is acceptable, non on what is breaking. In extreme, [any change is a breaking change](https://blogs.msdn.microsoft.com/ericlippert/2012/01/09/every-public-change-is-a-breaking-change/).